.semaphore/heroku-staging.yml

```diff
@@ -0,0 +1,29 @@
+version: v1.0
+name: Heroku deployment
```

Comment:
Maybe we can mention that this is the deployment for staging `Heroku deployment to staging`

Reply:
good point! Thank you. Done.